setup.cfg

```diff
@@ -38,24 +38,11 @@ install_requires =
   pytest-rerunfailures
   setuptools>=30.3.0
 packages = find:
-tests_require =
-  flake8>=3.6.0
-  flake8-blind-except
-  flake8-builtins
-  flake8-class-newline
-  flake8-comprehensions
-  flake8-deprecated
-  flake8-docstrings
-  flake8-import-order
-  flake8-quotes
-  mock
-  pep8-naming
-  pylint
-  pytest
-  pytest-cov
-  scspell3k>=2.2
 zip_safe = false
 
+[options.extras_require]
+test = flake8>=3.6.0; flake8-blind-except; flake8-builtins; flake8-class-newline; flake8-comprehensions; flake8-deprecated; flake8-docstrings; flake8-import-order; flake8-quotes; mock; pep8-naming; pylint; pytest; pytest-cov; scspell3k>=2.2
```

Comment:
Was there a specific motivation for the oneline approach versus something that matches the previous tests_require stanza?

```ini
[options.extras_require]
test =
 flake8>=3.6.0
 flake8-blind-except
 flake8-builtins
 flake8-class-newline
 flake8-comprehensions
...
```

seems easier to read and edit to me.

Reply:
> specific motivation

Blind following of a stack overflow post without much thought :no_mouth:

Reply:
Using newlines in 3cdb542